lib/client/api/sharing.js

```diff
@@ -175,17 +189,140 @@ module.exports = function (jiffClient) {
    */
   jiffClient.share_array = arraysSharing.jiff_share_array.bind(null, jiffClient);
 
+  /**
+   * Share an array of values. Each sender may have an array of different length. This is handled by the lengths parameter.
+   * This function will reveal the lengths of the shared array.
+   *
+   * If parties would like to keep the lengths of their arrays secret, they should agree on some "max" length apriori (either under MPC
+   * or as part of the logistics of the computation), all their arrays should be padded to that length by using appropriate default/identity
```

Comment:
please say explicitly that null and undefined do not work. They have to be values in [0, Zp)